source/jormungandr/jormungandr/instance.py

```diff
@@ -93,11 +98,14 @@ def _set_default_street_network_config(street_network_configs):
 
 
 class Instance(object):
+    name = None  # type: Text
```

Comment:
Why isn't that unicode?

Reply:
`Text` is `unicode` in python2 and `str` in python3

Reply:
Ah, so it's ambiguous in between and a new name but it's unicode all the way.
OK  :-|